koku/api/user_settings/views.py

```diff
@@ -29,6 +39,47 @@ def __init__(self, message):
         self.detail = {"detail": force_text(message)}
 
 
+@dataclass
+class SettingParamsHandler:
+    setting: str
+    retriever: object = field(init=False)
```

Comment:
```suggestion
    get_param: object = field(init=False)
```
`get_param` (or something similar) is closer to `update_param` than is retreiver.

Reply:
sure, I can update this.